heareval/predictions/task_predictions.py

```diff
@@ -784,18 +937,20 @@ def task_predictions_train(
     start = time.time()
     predictor: AbstractPredictionModel
     if metadata["embedding_type"] == "event":
-        validation_target_events = json.load(
-            embedding_path.joinpath("valid.json").open()
-        )
-        test_target_events = json.load(embedding_path.joinpath("test.json").open())
+        # Only validation target event is required while searching for best grid point
```

Comment:
Are you sure? Don't we want to include early stopping on validation when we're training the final model as well?